workos/events.py

```diff
@@ -24,6 +68,45 @@ def request_helper(self):
             self._request_helper = RequestHelper()
         return self._request_helper
 
+    def list_dsync_events(
+        self,
+        limit: Optional[int] = None,
+        organization_id: Optional[str] = None,
+        after: Optional[str] = None,
+        range_start: Optional[str] = None,
+        range_end: Optional[str] = None,
+    ) -> ListDsyncEventsResponse:
+        if limit is None:
+            limit = RESPONSE_LIMIT
+            default_limit = True
```

Comment:
Is `default_limit` used?

Reply:
It can get updated (look at other function), I removed it since it seemed unnecessary but didn't get to testing it to confirm